website/static/js/paginator.js

```diff
@@ -111,12 +111,16 @@ var Paginator = oop.defclass({
         }
     },
     nextPage: function(){
-        this.currentPage(this.currentPage() + 1);
-        this.fetchResults();
+        if (this.currentPage() !== this.numberOfPages() - 1) {
```

Comment:
This will prevent the `>` button from moving beyond the last page, but doesn't seem to give the user any indication of why, or that an action took place.

Instead or in addition to this logic, could we disable the `>` button so that it's apparent that it is inactive when there is no next page?